app/views/sample_types/_form.html.erb

```diff
@@ -18,6 +18,7 @@
     </div>
 
     <%= render partial: "projects/project_selector", locals: { resource: @sample_type } -%>
+    <%= render partial: 'assets/manage_specific_attributes', locals:{f:f} if show_form_manage_specific_attributes? %>
```

Comment:
for consistency, should use `assets/asset_form_bottom` with the relevant options passed